employees/views.py

```diff
@@ -334,6 +327,48 @@ def post(  # pylint: disable=unused-argument
         return self.redirect_to_another_month(request)
 
 
+@method_decorator(login_required, name="dispatch")
+@method_decorator(check_permissions(allowed_user_types=[CustomUser.UserType.ADMIN.name]), name="dispatch")
+class AuthorReportView(BaseAuthorReportView):
+    url_name = "author-report-list"
+    extra_context = {"only_projects_reports": False}
+
+    def get_queryset(self) -> QuerySet:
+        return CustomUser.objects.prefetch_related(
+            Prefetch(
+                "report_set",
+                queryset=Report.objects.filter(date__year=self.kwargs["year"], date__month=self.kwargs["month"]),
+            )
+        )
+
+
+@method_decorator(login_required(), name="dispatch")
+@method_decorator(
+    check_permissions(allowed_user_types=[CustomUser.UserType.ADMIN.name, CustomUser.UserType.MANAGER.name]),
+    name="dispatch",
+)
+@method_decorator(check_manager_permissions(), name="dispatch")
+class AuthorReportProjectView(BaseAuthorReportView):
+    url_name = "author-report-project-list"
+    extra_context = {"only_projects_reports": True}
```

Comment:
You can add it directly in `get_context_data` as you are overwriting it.

Reply:
Now I will not overwrite `get_context_data` so I will keep it.